code/modules/nano/nanoui.dm

```diff
@@ -106,14 +106,13 @@ nanoui is used to open and update nano browser uis
   */
 /datum/nanoui/proc/add_common_assets()
 	add_script("libraries.min.js") // A JS file comprising of jQuery, doT.js and jQuery Timer libraries (compressed together)
-	add_script("nano.js") // A JS file of the NanoUI JavaScript concatenated into one file.
-	add_stylesheet("shared.css") // this CSS sheet is common to all UIs
-	add_stylesheet("icons.css") // this CSS sheet is common to all UIs
-
-	//codemirror
-	add_script("codemirror-compressed.js") // A custom compressed JS file of codemirror, with CSS highlighting
-	add_stylesheet("codemirror.css") // this CSS sheet is common to all UIs, so all UIs can use codemirror
-	add_stylesheet("cm_lesser-dark.css") //CSS styling for codemirror, dark theme
+	add_script("nano.js")          // A JS file of the NanoUI JavaScript compressed into one file.
+	add_stylesheet("nanoui.css")   // Concatenated CSS sheet common to all UIs, contains all of the standard NanoUI styling.
+
+	// CodeMirror
+	add_script("codemirror-compressed.js") // A custom minified JavaScript file of CodeMirror, with the following plugins: CSS Mode, NTSL Mode, CSS-hint addon, Search addon, Sublime Keymap.
```

Comment:
Because I half-integrated CodeMirror into Nano while working on that, then never finished integrating it or using it anywhere. Still want to sometime, though.